website/content/docs/setup.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Setup
+description: Install Virtual Kubelet using one of several methods
+weight: 1
+---
+
+You can install Virtual Kubelet in one of several ways:
+
+* Using [`go install`](#go-install)
+* By building [from source](#source)
+
+## Using `go install` {#go-install}
```

Comment:
I think I'd prefer to leave off this part. This "works" now, but also ends up missing build tags, static binary support, etc.

Reply:
Done